cluster_zonal.tf

```diff
@@ -134,3 +134,18 @@ resource "google_container_node_pool" "zonal_pools" {
 
   depends_on = ["google_container_cluster.zonal_primary"]
 }
+
+resource "null_resource" "wait_for_zonal_cluster" {
```

Comment:
This should probably go in its own PR.

Reply:
See https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/46